docs/input/docs/getting-started/whymaybe.md

```diff
@@ -0,0 +1,30 @@
+---
+Order: 10
+Title: Why does BBT.Maybe exist?
+Description: Introduction to BBT.Maybe, what problems are solved and how it is distinctive from other implementations of functional option type.
+---
+
+## Make optional reference type values explicit
```

Comment:
... and america great again. We should use meaningful titles, instead of this page-wide 2nd level heading I'd rather have multiple headings above short paragraphs, for example

_intro text_
_Features_ (instead of "goals")
_Advantages over similar libraries_

Reply:
Adapted.